frappe/modules/import_file.py

```diff
@@ -114,6 +115,11 @@ def import_doc(docdict, force=False, data_import=False, pre_process=None,
 		pre_process(doc)
 
 	ignore = []
+	if doc.doctype not in single_doctype:
+		db_modified = frappe.db.get_values(doc.doctype, doc.name, ['modified','name'], as_dict=1)
+		if db_modified and doc.modified:
+			if doc.name == db_modified[0].get('name') and doc.modified <= get_datetime_str(db_modified[0].get('modified')):
+				return False
```

Comment:
There is already code to validate this in `import_file_by_path` - https://github.com/frappe/frappe/blob/c66b07930183ee6ff7598a686c5f744b6c435ff1/frappe/modules/import_file.py#L62

Can you move this piece of code to this `import_doc` function instead?

Reply:
yes, your right but why, it is reinserting Custom Field and property setter even there is no modification in the field and the modified date also not matched. if multiple developers working on fixtures some times fixtures not reflecting  even modified date is greater and even if we have 2 apps with fixtures we have to maintain both